app/controllers/api/v1/user_check_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::UserCheckController < ApplicationController
+  layout false
+
+  skip_before_filter :verify_authenticity_token, :only => :index
+
+  def index
+    render :json => {user: current_user}
```

Comment:
For future proofing in case sensitive info is added to current_user whitelist the current attributes to name, email, isAdmin, etc.